core/src/main/java/com/netease/arctic/table/TableProperties.java

```diff
@@ -257,6 +256,20 @@ private TableProperties() {
 
   public static final String OWNER = "owner";
 
+  /**
+   * table format related properties
+   */
+  public static final String TABLE_FORMAT = "table-format";
+  public static final String TABLE_FORMAT_MIXED_ICEBERG = "mixed-iceberg";
```

Comment:
Can we resue the enum `com.netease.arctic.ams.api.TableFormat` rather than defining too many property values?